spec/support/capybara.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Why not using this helper from [`solidus_support`](https://github.com/solidusio/solidus_support/blob/c552a2aa80f60c4964857d9b6401aede84e6ee6a/lib/solidus_support/extension/feature_helper.rb) instead? It already contains most of the code added in these supports files.